### ISSUE TITLE
Reduce build times & scale performance of Content API (CMS Team work to ship to VSP-Operations team)

### ISSUE THREAD

**Issue by cmaeng** [department-of-veterans-affairs/va.gov-cms#948]
## Background
Looking to reduce build times down to under a minute and scale performance of Content API by moving to Tome Sync.
### User Story or Problem Statement
As a content editor, I need my edited content to appear in production as soon as I click "publish".

### Affected users and stakeholders
* CMS editors

### Hypothesis
We believe that moving away from GraphQL and to tome sync will speed up the content build and deploy process from close to 10 minutes to under 5.

### Assumptions
* Once our work is done we will hand-off to the Platform Ops team to do E2E testing and determine when to go-live in production.

## Acceptance Criteria
* The front end build is using the content export system (CES) to consume the content API on production
* Tome sync is set up and bringing back build times of 5 minutes or less
* New build processes documented
* Edge cases put in to the backlog, groomed, and planned

## Possible tickets to create for this epic
- [ ] Usability testing (eg with a prototype or other design stimula)
- [ ] Implementation
- [ ] Documentation
- [ ] Metrics (How will we measure any improvements)

## Labels for  this issue
- [/] Issue type (red) (defaults to Epic)

**Comment by oksana-c** [comment]
closing in favor of #4396 due to deprecation of cms export
